Views/InputsView.swift

```diff
@@ -355,7 +361,40 @@ class InputsView: UIView, UITableViewDataSource, UITableViewDelegate, UISearchBa
     return assets
   }
 
+  func saveSeeds(registeredSeeds: [RegisteredSeed]) {
+
```

Comment:
I think we don't need empty line after func declaration, but we need after blocks like loop, conditions